index.js

```diff
@@ -473,3 +473,24 @@ module.exports.sync = (command, args, options) => {
 		timedOut: false
 	};
 };
+
+module.exports.fork = (filePath, args = [], options = {}) => {
+	const stdioOption = stdio.fork(options);
+
+	return execa(
+		options.execPath || process.execPath,
+		[
+			path.normalize(filePath),
```

Comment:
Is `path.normalize()` needed here? Core Node.js does not do it, so I am assuming Node.js `child_process` methods will handle the path correctly even if not normalized. Same thing with `execa()`.